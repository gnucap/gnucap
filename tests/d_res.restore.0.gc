spice

.param r=1

v1 1 0 sin amplitude=1 frequency=1
r1 1 0 r
r2 1 0 {r}

.print tran v(1)
.tran 1.1 trace=n

.print tran eiv(r*) v(1)
.tran 2 trace=a

.status notime
.end
