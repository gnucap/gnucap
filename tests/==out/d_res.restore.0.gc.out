#Time       v(1)      
 0.         0.        
 0.011      0.069059  
 0.022      0.13779   
 0.039967   0.24848   
 0.1283     0.72161   
 0.21664    0.97809   
 0.34283    0.83464   
 0.46903    0.19337   
 0.59522   -0.56323   
 0.72142   -0.9839    
 0.84761   -0.81773   
 0.97381   -0.16384   
 1.1        0.58777   
#Time       eiv(r1)    eiv(r2)    v(1)      
 1.1        0.58777    0.58777    0.58777   
 1.109      0.63256    0.63256    0.63256   
 1.2575     0.99887    0.99887    0.99887   
 1.4431     0.34979    0.34979    0.34979   
 1.6287    -0.72356   -0.72356   -0.72356   
 1.8144    -0.91929   -0.91929   -0.91929   
 2.        -1.2566n   -1.2566n   -1.2566n   
Gnucap   System status
iterations: op=0, dc=0, tran=26, fourier=0, total=97
transient timesteps: accepted=20, rejected=0, total=20
nodes: user=1, subckt=0, model=0, total=1
dctran density=100.0%, ac density=100.0%
